patroni/__main__.py

```diff
@@ -81,13 +81,34 @@ def ensure_unique_name(self) -> None:
             return
 
     def get_tags(self) -> Dict[str, Any]:
-        return {tag: value for tag, value in self.config.get('tags', {}).items()
-                if tag not in ('clonefrom', 'nofailover', 'noloadbalance', 'nosync') or value}
+        return {
+            tag: value for tag, value in self.config.get('tags', {}).items()
+            if tag not in (
+                'clonefrom',
+                'nofailover',
+                'failover_priority',
```

Comment:
should it really be a tag?

Reply:
I chose tag because I wanted to keep things as similar to existing `nofailover` implementation as possible. Happy to try and move elsewhere in the configuration if it makes more sense

Reply:
@hughcapet @matthbakeredb @barthisrael what do you think?

Reply:
I haven't looked at the code changes but from the initial proposal it sounds reasonable.